src/assets/css/styles.scss

```diff
@@ -12,6 +12,7 @@
 
 * {
   box-sizing: border-box;
+  overflow-wrap: anywhere;
```

Comment:
Note: this isn't supported on Safari

Reply:
🔔 https://github.com/mozilla/addons-frontend/issues/10289